### PR TITLE
Create ZK_Community_Hub

### PR DESCRIPTION
**Pull request by emchoos**
Hello!

Sorry for the delay. Our approved community hub post is here: https://forum.devcon.org/t/community-hub-proposal-zk-community-hub/773

Please let us know if there is anything we need to change, and we look forward to seeing you in person soon!

### CODE REVIEW

File: ZK_Community_Hub [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+DIP: TBD
+Title: ZK Community Hub
+Status: Draft
+Themes: Social, Community Involvement
+Tags: Event Production
```
Comment (franzihei) [inline ZK_Community_Hub:6]:
```suggestion
Tags: Community Hub, Event Production
```

File: ZK_Community_Hub [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Authors: michael@0xparc.org, remy@0xparc.org
+Resources Required: Seating for discussion groups, TV-sized screen, Power Outlets, Signage, Desks and tables, Whiteboards
```
Comment (franzihei) [inline ZK_Community_Hub:8]:
```suggestion
Resources Required: Seating for discussion groups, TV-sized screen, Power Outlet, Signage, Desks and tables, Whiteboard
```

Note that the Community Hub production guideline only includes one power outlet and one whiteboard. If you need more, please bring it! We recommend bringing extension cords / multi-plugs!

**Review by franzihei — CHANGES_REQUESTED** [review]
Two small things, otherwise looks great from my side!

File: ZK_Community_Hub [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+Status: Draft
+Themes: Social, Community Involvement
+Tags: Community Hub, Event Production
+Authors: michael@0xparc.org, remy@0xparc.org
+Resources Required: Seating for discussion groups, TV-sized screen, Power Outlet, Signage, Desks and tables, Whiteboard
+Discussion: https://forum.devcon.org/t/community-hub-proposal-zk-community-hub/773
+---
+
+-----Summary of Proposal-----
+__Simple Summary__
```
Comment (ligi) [inline ZK_Community_Hub:13]:
```suggestion
## Summary of Proposal

```

File: ZK_Community_Hub [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+The ZK Community Hub is a gathering point for various sub-communities in the applied ZK ecosystem. This includes communities focused around development and maintenance of proving system libraries, ZK circuit developers, ZK tooling, developers of ZK-enabled apps for either privacy or scalability, ZK rollup and zkEVM-focused teams, and more.
+
+-----Abstract-----
```
Comment (ligi) [inline ZK_Community_Hub:16]:
```suggestion
## Abstract
```

File: ZK_Community_Hub [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+Finally, the ZK Community Hub serves as an ad-hoc hangout spot or Schelling Point for ZK-focused attendees.
+
+We will invite teams from across the ecosystem, including:
+
+- zkEVM and rollup teams.
+- Teams building ZK infrastructure—proving stacks, libraries, etc.
+- ZK application development teams, such as PSE or 0xPARC’s identity application teams.
+
+-----Motivation & Rationale-----
```
Comment (ligi) [inline ZK_Community_Hub:33]:
```suggestion
## Motivation & Rationale
```

File: ZK_Community_Hub [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+- At the infrastructure layer, we will hold office hours where, as an example, a team building a ZK toolstack (for example, circom) may book a slot and invite community members to come by to ask the core development team questions.
+- At the L2 layer, we will host meet-and-greet sessions. For example, a rollup team (like Scroll, Hermez, zkSync, Starkware) may host third-party developers who are interested in developing on top of the rollup.
+- At the technical ecosystem layer, we’ll host fireside chats or round-table discussions. These would be guided discussions on specific topics (ZK Identity, ZK machine learning, ZK security), led by a domain expert where anyone at Devcon who is interested in the topic may participate in.
+- For those interested in consumer facing applications or those learning about ZK for the first time, we will have guided demos in informal sessions where community members can drop by the Community Hub to try out a ZK app / set of apps (try out HeyAnon, Zordle, EthDos, etc.), with guidance from a team member.
+
+-----Implementation-----
```
Comment (ligi) [inline ZK_Community_Hub:41]:
```suggestion
## Implementation
```

File: ZK_Community_Hub [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+- Additionally, we imagine that the space will be used informally for followups with speakers on the ZK track, or other meetups.
+    - Fo example, here are some talks / programming held elsewhere that the ZK Community Hub can be used to facilitate continued discussion for
+        - SLS - ZK Apps Demo Session + Panel
+        - SLS - PLONKish Day
+        - SLS - ZK Security Gathering
+        - SLS - PSE Projects
+        - Main talk - Dark Forest Talk
+        - Main talk - Recursive ZK Applications and Affordances
+
+-----Operational Requirements & Ownership-----
```
Comment (ligi) [inline ZK_Community_Hub:53]:
```suggestion
## Operational Requirements & Ownership
```

File: ZK_Community_Hub [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+__Please answer questions below:__
```
Comment (ligi) [inline ZK_Community_Hub:54]:
```suggestion

```

File: ZK_Community_Hub [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+Default layout will be perfect!
+
+2. Who will be responsible for the proposal to be implemented effectively? (i.e. working on Day 0)
+Michael, Remy, and gubsheep will take the lead.
+
+3. What other projects could this proposal be integrated with? (Bonus points for collaboration across teams :))
+We will be syncing up with catsnacks from the EF PSE team to make sure that there isn’t any overlap + we will be encouraging cross pollination of communities.
+
+
+-----Links & Additional Information-----
```
Comment (ligi) [inline ZK_Community_Hub:65]:
```suggestion
```

**Review by ligi — CHANGES_REQUESTED** [review]

File: ZK_Community_Hub [hunk 10]
```diff
@@ -0,0 +1,61 @@
+---
+DIP: TBD
```
Comment (ligi) [inline ZK_Community_Hub:2]:
```suggestion
DIP: 28
```

**Pull request merged**